### PR TITLE
Make templates work with more customers

### PR DESCRIPTION
**Pull request by CalleB3**

### CODE REVIEW

File: azure/aks/variables.tf [hunk 1]
```diff
@@ -216,3 +197,14 @@ variable "external_dns_hostname" {
   type        = string
   default     = ""
 }
+
+variable "aks_name_suffix" {
```
Comment (NissesSenap) [inline azure/aks/variables.tf:201]: Would it be possible to create a third option of having a aks.tf or something like that which contain null.

File: azure/aks/variables.tf [hunk 2]
```diff
@@ -121,24 +120,6 @@ variable "opa_gatekeeper_config" {
   }
 }
 
-variable "ingress_config" {
```
Comment (NissesSenap) [inline azure/aks/variables.tf:124]: What is the reasoning of making this in to a local?

**Review by NissesSenap — COMMENTED** [review]
A few comments

File: azure/aks/variables.tf (hunk 2 above)
Comment (CalleB3) [reply azure/aks/variables.tf:124]: This is due to this customer using very custom config and using variables in the code. So using variable in a variable doesnt work. Hence the local being used.

File: azure/aks/variables.tf (hunk 1 above)
Comment (CalleB3) [reply azure/aks/variables.tf]: Doesnt work, providers are duplicated = issue

**Review by NissesSenap — APPROVED** [review]

**Comment by NissesSenap** [comment]
LGTM

**Review by jimgus — CHANGES_REQUESTED** [review]

**Pull request merged**